src/settings.py

```diff
@@ -114,20 +114,39 @@ def get_version(settings_filename):
         print("Error reading version")
         return ""
 
+def set_version(settings_filename, version):
+    try:
+        with open(settings_filename, "r") as settings_file:
+            settings = yaml.safe_load(settings_file)
+
+        settings["common"]["version"] = version
+
+        with open(settings_filename, "w") as settings_file:
+            yaml.dump(settings, settings_file, default_flow_style=False)
+
+    except Exception as e:
+        print("Error writing version")
+
 def paths_transfer_in_settings(initial_settings_filename, final_settings_filename):
     with open(initial_settings_filename, "r") as settings_file:
         initial_settings = yaml.safe_load(settings_file)
 
     with open(final_settings_filename, "r") as settings_file:
         final_settings = yaml.safe_load(settings_file)
 
-        for k, v in initial_settings.items():
-            if k in final_settings:
-                final_settings[k] = v
+        compare_settings(initial_settings, final_settings)
 
         with open(final_settings_filename, "w") as settings_file:
             yaml.dump(final_settings, settings_file, default_flow_style=False)
 
+def compare_settings(settings1, settings2):
+    for key in set(settings1) & set(settings2):
```

Comment:
hmm, this part will skip keys that do not exist in the final settings, right? But we should add new settings too.